README.md

```diff
@@ -71,6 +71,9 @@ This is just a simple delegating method that explicitly calls
 Futures.transform(future, Function). There is also a corresponding
```

Comment:
This line doesn't seem to be correct any longer, IIUC? The call is instead to `Futures.transformAsync`?